### ISSUE TITLE
Discord Parameter example incomplete

### ISSUE THREAD

**Issue by avluis** [BtoBastian/sdcf4j#2]
First off, great library you have here -- I've been looking for something that emulates Discord.Net.Commands for Java.

On to the issue -- your example of a dynamic command (I'm using JDA) is a bit incomplete:
```java
@Command(aliases = {"!channelInfo", "!ci"}, description = "Pong!", async = true, privateMessages = false)
public String onCommand(Channel channel) {
    return "You are in channel #" + channel.getName() + " with id " + channel.getId();
}
```

As is, you will encounter a `NullPointerException` as it won't be able to resolve `Channel` -- this works instead:
```java
@Command(aliases = {"!channelInfo", "!ci"}, description = "Pong!", async = true, privateMessages = false)
public String onCommand(MessageReceivedEvent event) {
    Channel channel = event.getTextChannel();
    return "You are in channel #" + channel.getName() + " with id " + channel.getId();
}
```

I've only tested with JDA, so I cannot speak for the other libraries -- this is under `JDA 2.2.1_353`.

**Comment by BtoBastian** [comment]
Hey, thank you.
The example should work fine - if it doesn't I messed up something with JDA. I'll check it.

**Comment by hgjkant** [comment]
I have found the same. Arguments of type `Channel` and `Message` are null.

**Comment by BtoBastian** [comment]
Fixed in 1.0.3
https://github.com/BtoBastian/sdcf4j/commit/39c363be27d23e6ae2bbc43a747587700a78ca55
